src/index.js

```diff
@@ -25,6 +25,13 @@ async function handleVersioningByExtension(ext, file, versionPath, releaseType)
 
 async function run() {
   try {
+
+    process.on('unhandledRejection', (reason, promise) => {
```

Comment:
Maybe nicer to add this all the way at the bottom before `run` is called? Also maybe just pass the error directly to `console.log` and `core.setFailed`?

Reply:
Moved.
Passing reason directly to the `console.log` and `core.setFailed` will result in casting it to a simple string and loosing stack trace. I suppose it is not the best option.